ouroboros-consensus-cardano/tools/db-analyser/Analysis.hs

```diff
@@ -79,15 +80,16 @@ data AnalysisEnv blk = AnalysisEnv {
     , db         :: Either (ImmutableDB IO blk) (ChainDB IO blk)
     , registry   :: ResourceRegistry IO
     , ledgerDbFS :: SomeHasFS IO
+    , limit      :: Limit
```

Comment:
`limit` is a bit terse, for the name in the code and for the name of the command-line option.

Maybe `NumBlocksToProcess` and `--num-blocks-to-process`?

Reply:
when you run the tool you get meta info what you are limiting

```
Usage: db-analyser --db PATH [--verbose]
                   (....)
                   [--limit NUMBER OF BLOCKS]
 ```
 
TBH I much prefer using a tool where I have to type `--limit` then `--num-blocks-to-process`. It's shorter and easy to remember. `--limit` I can type, `--num-blocks-to-process` I will most likely copy-paste from help message